src/jit/codegenxarch.cpp

```diff
@@ -1813,162 +1813,20 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                     assert(structDesc.eightByteCount == CLR_SYSTEMV_MAX_EIGHTBYTES_COUNT_TO_PASS_IN_REGISTERS);
 
                     regNumber retReg0 = REG_NA;
-                    emitAttr size0 = EA_UNKNOWN;
-                    unsigned offset0 = structDesc.eightByteOffsets[0];
+                    unsigned __int8 offset0 = 0;
```

Comment:
If i understand correctly, this is a case of returning a struct in register (RAX/XMM0 or RAX/XMM0 and RDX/XMM1).
When i looked in lowerxarch.cpp i don't see useCandidates being specified for a struct returned in two registers.